contracts/AaveV2StablecoinCellar.sol

```diff
@@ -178,7 +270,7 @@ contract AaveV2StablecoinCellar is IAaveV2StablecoinCellar, ERC20, Ownable {
      */
     constructor(
         ERC20 _asset,
-        address[] memory _approvedPositions,
+        ERC20[] memory _approvedPositions,
```

Comment:
When I've been deploying the cellar, I've been aware of this too and just didn't include `_asset` in _approvedPositions` because you are right it would be redundant.